package.json

```diff
@@ -156,8 +156,10 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.3.4",
+    "@date-io/date-fns": "1",
```

Comment:
What is this version ? Can't we have a more recent version ?

Reply:
That's why: [Do not update if you are using @material-ui/pickers v3](https://github.com/dmtrKovalenko/date-io/releases/tag/v2.0.0)

Reply:
Maybe to add in the commit message